frontend/src/Router.tsx

```diff
@@ -0,0 +1,20 @@
+import { Route,Routes } from 'react-router-dom'
+import Homepage from './Homepage'
+import Login from './Login'
+import Nav from './Nav'
+import Signup from './Signup'
+const Router = () => {
+ 
+    return (
+  <>
+  <Nav/>    
+  <Routes>
+  <Route path='/' element={<Homepage/> } />
+  <Route path='/login' element={<Login/>} />
+  <Route path='/signup' element={<Signup/>} />
```

Comment:
```suggestion
  <Route path='/signup' element={<Signup/>} />
 
```
don't try to use magical string as we are giving /signup here , we should use some variable here.